example/c/main.c

```diff
@@ -33,9 +33,7 @@ int main(int argc, char *argv[]) {
 
     print_decimal(fd);
 
-    DiplomatStringView bn = {"bn", 2};
-
-    Locale* locale = icu4x_Locale_new_mv1(bn);
+    Locale* locale = icu4x_Locale_new_mv1((DiplomatStringView){.data = "bn", .len = 2});
```

Comment:
does this really need the cast? 😕

Reply:
it fails to _parse_ without the cast. I don't understand it either.

C is weird and has terrible errors, if you can figure out how to avoid the cast be my guest 🙃🙃

Reply:
ew. in that case I'm not sure if the inline is actually nicer 🤷

Reply:
yeah feel free to change it back if you'd like. I'm very ambivalent here